defaults/main.yml

```diff
@@ -59,3 +59,9 @@ podman_containers_conf: {}
 # __podman_registries_conf_system or __podman_registries_conf_user
 # paths, respectively, depending on execution mode selected
 podman_registries_conf: {}
+
+# containers-storage.conf(5) settings, these will be provided in a drop-in
```

Comment:
```suggestion
# containers-storage.conf(5) settings, these will be provided in a drop-in
```

Probably a copy-paste:  storage.conf does not provide drop-in support.

Reply:
Ah ok, then that's going to get complicated. I'll have to rethink this and follow the pattern that the ssh system role did.